client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -56,38 +56,43 @@ const HostingFeatures = () => {
 			? `/hosting-config/${ siteId }`
 			: `/overview/${ siteId }`
 	);
+	const hasRedirectQueryParam = searchParams.has( 'redirect_to' );
 	const hasEnTranslation = useHasEnTranslation();
 
-	const { data: siteTransferData, refetch: refetchSiteTransferData } = useSiteTransferStatusQuery(
-		siteId || undefined
-	);
+	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId || undefined, {
+		refetchInterval: 3000,
+		refetchIntervalInBackground: true,
```

Comment:
Could you remind me which bug this fixes, @nightnei?

Reply:
It's just refactoring which you asked to consider [here](https://github.com/Automattic/wp-calypso/pull/93488#discussion_r1718390273). So I removed manual refetch and moved to `refetchInterval ` and `refetchIntervalInBackground `.